sass/directives/_carousels.scss

```diff
@@ -88,6 +88,63 @@
   }
 }
 
+@mixin resource-header {
+  h2, p, h3, h1 {
+    display: inline-block;
+  }
+
+  nav {
+    display: inline-block;
+    float: right;
+  }
+}
+
+@mixin resource-carousel-arrow {
+  z-index: z($z-context, page_content, $page_content, carousel__arrow);
+  color: $inactive-carousel-button-color;
+  font-size: $direction-arrow-size / 2;
+  cursor: pointer;
+  border: 1px solid $inactive-carousel-button-color;
+  padding: $base-padding / 2;
+
+  &:hover,
+  &:active,
+  &:focus {
+    color: $active-carousel-button-color;
+    border: 1px solid $active-carousel-button-color;
+  }
+
+  &--left {
+    margin-right: 1px;
+  }
+}
+
+@mixin resource-continue-block {
+  border-top: $base-border;
+  padding-top: $base-padding / 2;
+  padding-bottom: $base-padding / 2;
+  padding-left: 1em;
+  font-size: $h3-font-size;
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  margin-top: $base-padding;
+
+  .fa {
```

Comment:
As mentioned here, icons can go in CSS, like the one below this one: https://github.com/cbdr/employer/pull/685/files#r103507701